### PR TITLE
Update objectnav_robothor_rgb_resnetgru_ddppo.py

### PR DESCRIPTION
**Pull request by ekolve**

### CODE REVIEW

**Comment by ekolve** [comment]
I noticed in some of modules ObjectNavRoboThorRGBPPOExperimentConfig and other modules the name ObjectNavThorRGBPPOExperimentConfig, so I'm not sure what name you want to use across all of them.

File: projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 )
 
 
-class ObjectNaviThorRGBPPOExperimentConfig(
+class ObjectNavThorRGBPPOExperimentConfig(
```
Comment (jordis-ai2) [inline projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo.py:16]: I guess it should be `ObjectNavRoboThorRGBPPOExperimentConfig`. I thought we had gone through all of these things, but good that you double-checked 👍

Comment (ekolve) [reply projects/objectnav_baselines/experiments/robothor/objectnav_robothor_rgb_resnetgru_ddppo.py]: Updated to ObjectNavRoboThorRGBPPOExperimentConfig

**Comment by jordis-ai2** [comment]
LGTM

**Pull request merged**